package.json

```diff
@@ -1,12 +1,25 @@
 {
-  "version": "0.1.0",
+  "version": "0.0.0",
   "license": "MIT",
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "files": [
-    "dist",
-    "src"
+    "dist"
   ],
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    },
+    "./button": "./dist/button.js",
+    "./tag": "./dist/tag.js"
+  },
+  "typesVersions": {
+    "*": {
+      "button": ["dist/button.d.ts"],
+      "tag": ["dist/tag.d.ts"]
+    }
+  },
```

Comment:
In the meantime, this should work for specifying the exact path for the component `.js` file. This will need to be part of the documentation unless there is simple fix for importing individual `.js` files.